pkg/clusters/addons/knative/knative.go

```diff
@@ -27,12 +27,17 @@ const (
 
 	// DefaultNamespace indicates the default namespace this addon will be deployed to.
 	DefaultNamespace = "knative-serving"
+
+	// DefaultVersion is the Knative version deployed when the user requests no specific version
+	DefaultVersion = "knative-v1.1.1"
```

Comment:
we have a mixture of addons either having a default version or deriving a base version from github releases, e.t.c.

Now that knative is at V1, it seems like perhaps we should just use the existing utility function to base our default/base version off of the latest release?

Reply:
92056b9